lib/ts/recipe/session/with-jwt/recipeImplementation.ts

```diff
@@ -144,25 +158,23 @@ export default function (
             sessionHandle: string;
             newAccessTokenPayload: any;
         }): Promise<void> {
-            let sessionInformation = await this.getSessionInformation({ sessionHandle });
-
-            let jwtPropertyName = config.jwt.propertyNameInAccessTokenPayload;
-
-            if (config.jwt.getPropertyNameFromAccessTokenPayload !== undefined) {
-                jwtPropertyName = config.jwt.getPropertyNameFromAccessTokenPayload(
-                    sessionInformation.accessTokenPayload
-                );
+            if (newAccessTokenPayload[ACCESS_TOKEN_PAYLOAD_JWT_PROPERTY_NAME_KEY] !== undefined) {
+                throw new Error(JWT_RESERVED_KEY_USE_ERROR_MESSAGE);
             }
```

Comment:
Remove this throwing from here and createNewSession. We need to remove it here cause the value of `newAccessTokenPayload` can be based on the result of `getAccessTokenPayload` which will contain the `_jwtPName`. 

If we are removing it here, we should just remove it from createNewSession too cause that won't make much of a diff then.. we can override user's provided value with ours.

Reply:
Done